frontend.py

```diff
@@ -44,33 +60,58 @@ def load_robots(state):
     robot_sprites = []
     # Only active robots will be drawn.
     for robot in state.get_active_robots():
-        robot_sprite = create_sprites(robot.coordinates, [robot])
+        robot_sprite = create_robot_sprites(robot.coordinates, [robot])
         robot_sprites.extend(robot_sprite)
     return robot_sprites
 
 
-def create_sprites(coordinate, items):
+def create_sprites(coordinate, tiles):
     """
     Return list of sprites of items.
 
-    coordinate: coordinate of tiles or robot
-    items: a list of Tile or Robot objects
+    coordinate: coordinate of tiles
+    tiles: a list of Tile
+    """
+    tiles_sprites = []
+    for tile in tiles:
+        rotation = tile.direction.value
+        path = tile.path
+        x, y = coordinate
+        img = pyglet.image.load(path)
+        img.anchor_x = img.width//2
+        img.anchor_y = img.height//2
+        tile_x = x*TILE_WIDTH
+        tile_y = y*TILE_HEIGHT
+        img_sprite = pyglet.sprite.Sprite(img, x=img.anchor_x + tile_x,
+                                               y=img.anchor_y + tile_y)
+        img_sprite.rotation = rotation
+        tiles_sprites.append(img_sprite)
+    return tiles_sprites
+
+
+def create_robot_sprites(coordinate, robots):
+    """
+    Return list of sprites of robots.
+    coordinate: coordinate of tiles
+    robots: a list of robots
     """
-    items_sprites = []
-    for item in items:
-        rotation = item.direction.value
-        path = item.path
+    robots_sprites = []
+    robot_paths = get_robot_paths
+    
+    for robot in robots:
+        rotation = robot.direction.value
+        path = robot_paths
         x, y = coordinate
         img = pyglet.image.load(path)
```

Comment:
Když načítáš, řekněne, obrázek pro robota `mintbot`, jak vypadá cesta k obrázku?

Reply:
předpokládám že `/img/robots/png/mintbot.png'

Reply:
teď mě mate když si to vyprintuju ty cesty, proč u těch front_path není taky první lomítko `/` 
[(WindowsPath('img/robots_map/png/hanka.png'), './img/robots/png/hanka.png'),....]

Reply:
> předpokládám že `/img/robots/png/mintbot.png'`

Takže nepotřebuješ `path = robot_paths`, ale můžeš `path` vždycky nastavit podle jména robota.

> proč u těch front_path není taky první lomítko

Přesně tak, první lomítko tam nemá být.

Reply:
takže ale stejně potřebuju napsat funkci kde ty cesty k tomu jménu nějak přiřadím.

Reply:
Potřebuješ zjistit cestu pro konkrétního robota, a místo:
```
path = robot_paths
```
nastavit tu `path` na tuhle cestu. Například pro robota jménem `hanka` je cesta `img/robots_map/png/hanka.png`.

Reply:
jo jenže každý robot má dvě cesty k obrázkům, to mě mate.

Reply:
Tady jsi ve `frontend.py`, kde se zobrazuje jen mapa. Ty velké obrázky se tu nevyužijí.

Reply:
jenže v tomto v případě vlastně asi řeším jen tu jednu cestu, ta druhá se udělá v interface že?